backend/Gemfile

```diff
@@ -39,7 +39,7 @@ group :development, :test do
   gem 'bullet'
   gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
 end
-
+#
```

Comment:
remove this?

Reply:
Oh, that was probably when writing in live share, will fix this!